app/views/pages/home.html.erb

```diff
@@ -3,6 +3,25 @@
     <div class="govuk-main-wrapper">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
+          <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+            <div class="govuk-notification-banner__header">
+              <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                Important
+              </h2>
+            </div>
+            <div class="govuk-notification-banner__content">
+              <p class="govuk-notification-banner__heading">
+                It may take slightly longer to be assigned an adviser between
+                24 December 2021 and the 2 January 2022.
+              </p>
+
+              <p>
```

Comment:
Do we need this para? I don't think it is necessary - we can just have the banner heading.

If it is mandatory to have body copy, this will need rewording